packages/repl/src/lib/Repl.svelte

```diff
@@ -196,8 +206,6 @@
 			$module_editor?.clearEditorState();
 		}
 
-		EDITOR_STATE_MAP.set(filename, $module_editor?.getEditorState());
```

Comment:
Also here we don't need this anymore because every file get's his entry in the EDITOR_STATE_MAP either when is set at the beginning or upon creation/modification

Reply:
Amazing!!